src/esp/physics/bullet/BulletBase.h

```diff
@@ -135,10 +135,9 @@ class BulletBase {
   std::shared_ptr<btMultiBodyDynamicsWorld> bWorld_;
 
   /** @brief Static data: All components of a @ref RigidObjectType::SCENE are
-   * stored here. See @ref btCollisionObject.  Also, all objects set to STATIC
-   * are stored here.
+   * stored here. Also, all objects set to STATIC are stored here.
    */
-  std::vector<std::unique_ptr<btCollisionObject>> bStaticCollisionObjects_;
+  std::vector<std::unique_ptr<btRigidBody>> bStaticCollisionObjects_;
```

Comment:
Duplicating a PR that went into master: https://github.com/facebookresearch/habitat-sim/pull/814

Reply:
@eundersander Pull from master to make it disappear on the files changed tab.

Reply:
Are you suggesting rebasing onto master? That's a bigger project than I'd like to tackle here.